lisp/mastodon-tl.el

```diff
@@ -108,9 +139,16 @@ Don't move if nothing else to move to is found, i.e. near the end of the buffer.
   "Move to the previous interesting item.
 
 This could be the previous toot, link, or image; whichever comes first.
-Don't move if nothing else to move to is found, i.e. near the start of the buffer."
+Don't move if nothing else to move to is found, i.e. near the start of the buffer.
+This also skips tab items in invisible text, i.e. hidden spoiler text."
   (interactive)
-  (let ((next-range (mastodon-tl--find-next-or-previous-property-range 'mastodon-tab-stop (point) t)))
+  (let (next-range
+        (search-pos (point)))
+    (while (and (setq next-range (mastodon-tl--find-next-or-previous-property-range 'mastodon-tab-stop search-pos t))
```

Comment:
*nit can we set a line limit between 80 and 90 (with some exemptions)? This does not have to be done for this commit. Perhaps when we are refactoring later we can systematically shorten some of the lines in `mastodon-tl.el`.